pkgs/servers/ursadb/default.nix

```diff
@@ -1,40 +1,31 @@
-{ lib, stdenv, fetchFromGitHub, cmake, zeromq, cppzmq }:
+{ lib, stdenv, fetchFromGitHub, cmake }:
 
 stdenv.mkDerivation rec {
   pname = "ursadb";
-  version = "1.2.0";
+  version = "1.5.0";
 
   src = fetchFromGitHub {
     owner = "CERT-Polska";
     repo = "ursadb";
     rev = "v${version}";
-    hash = "sha256-/EK1CKJ0IR7fkKSpQkONbWcz6uhUoAwK430ljNYsV5U=";
+    hash = "sha256-JssL1Yd+2CPy9fDPJnstiRucdy6TyUdb0Q6xa160CLM=";
+    fetchSubmodules = true;
   };
 
   postPatch = ''
     substituteInPlace CMakeLists.txt \
-      --replace \
-        "add_executable(ursadb_test Tests.cpp)" "" \
-      --replace \
-        "target_link_libraries(ursadb_test ursa)" ""
-  '';
-
-  installPhase = ''
-    mkdir -p $out/bin
-    cp ursadb $out/bin/
-    cp ursadb_new $out/bin/
-    cp ursadb_trim $out/bin/
+      --replace "add_executable(ursadb_test src/Tests.cpp)" "" \
+      --replace "target_link_libraries(ursadb_test ursa)" "" \
+      --replace "target_enable_ipo(ursadb_test)" "" \
+      --replace "target_clangformat_setup(ursadb_test)" "" \
+      --replace 'target_include_directories(ursadb_test PUBLIC ${"$"}{CMAKE_SOURCE_DIR})' "" \
+      --replace "ursadb_test" ""
   '';
 
   nativeBuildInputs = [
     cmake
   ];
 
-  buildInputs = [
-    zeromq
-    cppzmq
-  ];
-
```

Comment:
```suggestion
  passthru.tests = {
    unittest = runCommand "${finalAttrs.pname}-unittest" {} ''
      ${finalAttrs.finalPackage}/bin/ursadb_test > $out
      if [[ `cat $out` != *"All tests passed"*  ]]; then
        echo "Error: test failed"
        exit 1
      fi
    '';
    bench = runCommand "${finalAttrs.pname}-bench" {} ''
      ${finalAttrs.finalPackage}/bin/ursadb_bench > $out
    '';
  };

```

Reply:
Looks like ursadb_test doesn't build with a new glibc version (due to something changing from a compile-constant to a variable). I've created an issue upstream to fix that: https://github.com/CERT-Polska/ursadb/issues/189. The fix will be applied in the next release (v1.6.0), so I think we can merge it as it is and add the tests later.

Reply:
👍